src/components/StatueRoutes.spec.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
I believe this won't be needed if the file extension is changed to jsx.

Reply:
Tried this and no dice unfortunately, but have removed the rule ignore.